api-extractor/src/ResolvedApiItem.ts

```diff
@@ -75,12 +79,16 @@ export default class ResolvedApiItem {
     deprecatedMessage: IDocElement[],
     isBeta: boolean,
     params:  {[name: string]: IParam},
-    returnsMessage: IDocElement[]) {
+    returnsMessage: IDocElement[],
+    apiTag: ApiTag,
+    apiItem?: ApiItem) {
```

Comment:
Since it's a private constructor, it's probably better top make it `ApiItem|undefined` so we don't forget to provide it